templates/outline.yaml

```diff
@@ -180,7 +180,7 @@ spec:
 
 ---
 
-apiVersion: networking.k8s.io/v1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
Please use the current stable ingress api version as before instead of a downgrade to a beta